modules/dmrpp_module/build_dmrpp_h4/build_dmrpp_util_h4.cc

```diff
@@ -233,16 +234,18 @@ int read_chunk(int sdsid, SD_mapping_info_t *map_info, int *origin)
 
 } /* read_chunk */
 
-string get_sds_fill_value_str(int32 sdsid, int32 datatype) {
+string get_sds_fill_value_str(int32 sdsid, int32 datatype, bool &dmrpp_h4_error, string &err_msg) {
 
     intn emptySDS = 0;
+    string ret_value;
     
     if (SDcheckempty(sdsid,&emptySDS) == FAIL) {
         SDendaccess(sdsid);
-        throw BESInternalError("SDcheckempty fails. ",__FILE__,__LINE__);
+        dmrpp_h4_error = true;
+        err_msg = "SDcheckempty fails at "+string(__FILE__) +":" + to_string(__LINE__);
+        return ret_value;
```

Comment:
Is the intent here to return "" on error?

Reply:
Yes, since dmrpp_h4_error is set to true, the caller will handle this as an error. The ret_value doesn't matter since it will not be used.